guide/src/main/asciidoc/problems/resourceNotFound.adoc

```diff
@@ -23,10 +23,10 @@ Content-Type: application/problem+json
 }
 ```
 
-Either the path parameters don't resolve to an existing resource. Look for the more details in the `issues` property.
+Either a path parameter or an optional child resource path don't resolve to an existing resource. Look for the more details in the `issues` property.
```

Comment:
don't -> doesn't? https://www.grammar-monster.com/lessons/either_or_plural_or_singular_verb.htm

Look for ~~the~~ more details